pkg/terraform/exec/plugins/aws.go

```diff
@@ -11,5 +11,9 @@ func init() {
 			ProviderFunc: aws.Provider,
 		})
 	}
-	KnownPlugins["terraform-provider-aws"] = exec
+	KnownPlugins["terraform-provider-aws"] = &TFPlugin{
+		Name:    "aws",
+		Exec:    exec,
+		Version: GetAwsVersion(),
```

Comment:
It is not clear to me whether it is important that we use the real version here, since we are not setting a version requirement in the `required_providers` block in the tf files. What happens if we just set all of the versions to v1.0.0 without prejudice?

Reply:
If we set the version to 1.0.0, terraform will bomb out stating that version is wrong.

Reply:
That's a wonderful reason for setting the real version. :)

Reply:
And, in case the (unauthenticated) is of concern, even when we set the "correct" version, it is still unauthenticated.
```
time="2021-03-18T21:45:56Z" level=debug msg="- Installed openshift/local/aws v3.1.0 (unauthenticated)"
```